packages/cli/test/dev/integration-5.test.ts

```diff
@@ -447,3 +447,13 @@ test(
     }
   })
 );
+
+test(
+  '[vercel dev] should set charset in content-type header',
+  testFixtureStdio('test-tomato', async (_: any, port: any) => {
+    const response = await fetch(`http://localhost:${port}/`);
+    expect(response.headers.get('content-type')).toBe(
+      'text/html; charset=utf-8'
```

Comment:
Can we add this assertion to an existing test? Otherwise this is going to be much slower to add a new deployment here.